.gitignore

```diff
@@ -92,3 +94,4 @@ hs_err_pid*
 
 
 # End of https://www.gitignore.io/api/java,intellij
+/target/**
```

Comment:
should be target/**, look on last file from this PR, it's file from target directory